target_chains/ethereum/contracts/forge-test/Pyth.t.sol

```diff
@@ -12,156 +12,123 @@ import "./utils/WormholeTestUtils.t.sol";
 import "./utils/PythTestUtils.t.sol";
 import "./utils/RandTestUtils.t.sol";
 
-contract PythTest is Test, WormholeTestUtils, PythTestUtils, RandTestUtils {
+contract PythTest is Test, WormholeTestUtils, PythTestUtils {
```

Comment:
This is great that you've refactored the tests here to be with Accumulator. But I'm wondering whether they were already being tested in Pyth.WormholeMerkleAccumulator.t.sol. If they are already covered I think we can just remove the tests here.

Reply:
There is some overlap between tests in `Pyth.t.sol` and `Pyth.WormholeMerkleAccumulator.t.sol`, but some tests are different.

There are similar tests for:

- testParsePriceFeedUpdatesWorks
- testParsePriceFeedUpdatesWorksWithRandomDistinctUpdatesInput
- testParsePriceFeedUpdatesRevertsIfUpdateFeeIsNotPaid
- testParsePriceFeedUpdatesRevertsIfPriceIdNotIncluded
- testParsePriceFeedUpdateRevertsIfPricesOutOfTimeRange
- testParsePriceFeedUpdatesLatestPriceIfNecessary

And these are the tests that are only in `Pyth.t.sol`:

- testParsePriceFeedUpdatesWorksWithOverlappingWithinTimeRangeUpdates
- testParsePriceFeedUpdatesWorksWithOverlappingMixedTimeRangeUpdates
- testParsePriceFeedUpdatesRevertsIfUpdateVAAIsInvalid
- testParsePriceFeedUpdatesRevertsIfUpdateSourceChainIsInvalid

Should I remove the duplicated tests? (I would still prefer to keep `testParsePriceFeedUpdatesWorks` just to make it easier to debug the `setUp()` if it ever breaks.)